proof/refine/X64/CSpace1_R.thy

```diff
@@ -1752,6 +1753,7 @@ qed
 definition pspace_relations where
   "pspace_relations ekh kh kh' \<equiv> pspace_relation kh kh' \<and> ekheap_relation ekh kh'"
 
+(* unsure *)
```

Comment:
This is here because we should update it with what we rename `set_cap_not_quite_corres` to